### ISSUE TITLE
Submissions not being sent to Constant Contact

### ISSUE THREAD

**Issue by BlueprintMkt** [katzwebservices/Contact-Form-7-Newsletter#64]
Hello, 
In looking through the past threads, it looks like other users have had this issue and that the solution was an updated version of the plugin. However, I have the latest version of everything (I'm building a new site right now and just downloaded the latest versions of WordPress, Contact Form 7 Newsletter and all other plugins) and I'm still having this issue. 

I'm currently on WordPress v 4.3.1 and Contact Form 7  v. 4.3.1, too. I have everything set up so that it all looks great both on the font end and back end. When I submit a form, I get the emails generated by the plugin (both as the web admin and as the subscriber). However, when I get into Constant Contact, the submission is not there. I have tested this several times throughout the day, as have a couple other users, and we have had no success. 

Please let me know what to do. Thanks!

**Comment by aengus123** [comment]
Hi,

Checking in to confirm that after two days or so of troubleshooting, I am having same issue as BlueprintMkt. Everything seems to be working ok, its pulling in the lists etc, but for some reason the contacts just not getting added to the constant contact list. I've tried rolling back to previous versions of both the Contact Form 7 Newsletter plugin, Wordpress and Contact Form 7 (4.2 initial release) but I'm still not getting anywhere with it.

I know this plugin worked up until a few months ago, as a site I built previously that uses worked ok on launch, but is now failing...I guess its combination of Wordpress core and Cf7 updates have knocked something out of whack.

I'm struggling to debug this issue - if there is anything I can do from this end to help diagnose the problem with the plugin author, I'm more than happy to help :)
